examples/alerts/alerts.md

```diff
@@ -236,39 +236,26 @@ rules:
 
 ## Sidecar
 
-[//]: # "TODO(kakkoyun): Generate sidecar rules using thanos-mixin."
-<!-- [embedmd]:# (../tmp/thanos-sidecar.rules.yaml yaml) -->
+[embedmd]:# (../tmp/thanos-sidecar.rules.yaml yaml)
 ```yaml
+name: thanos-sidecar.rules
+rules:
 - alert: ThanosSidecarPrometheusDown
-  expr: thanos_sidecar_prometheus_up{name="prometheus"} == 0
-  for: 5m
-  labels:
-    team: TEAM
   annotations:
-    summary: Thanos Sidecar cannot connect to Prometheus
-    impact: Prometheus configuration is not being refreshed
-    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
-    dashboard: SIDECAR_URL
-- alert: ThanosSidecarBucketOperationsFailed
-  expr: rate(thanos_objstore_bucket_operation_failures_total{name="prometheus"}[5m]) > 0
+    message: Thanos Sidecar {{$labels.job}} {{$labels.pod}} cannot connect to Prometheus.
+  expr: |
+    sum by (job, pod) (thanos_sidecar_prometheus_up{job=~"thanos-sidecar.*"} == 0)
   for: 5m
   labels:
-    team: TEAM
+    severity: critical
+- alert: ThanosSidecarUnhealthy
   annotations:
-    summary: Thanos Sidecar bucket operations are failing
-    impact: We will lose metrics data if not fixed in 24h
-    action: Check {{ $labels.kubernetes_pod_name }} pod logs in {{ $labels.kubernetes_namespace}} namespace
-    dashboard: SIDECAR_URL
-- alert: ThanosSidecarGrpcErrorRate
```

Comment:
@daixiang0 Let's not lose this alert, we have similar more up-to-date ones in other components' alerts.

Reply:
Then we're good to merge.

Reply:
The impact and action part are only seen in this alert, is it necessary to add for all component alert?

Reply:
No, I'd say it's not necessary. At this stage, these are just examples to start from.